README.md

```diff
@@ -33,7 +33,16 @@ node {
 }
 ```
 
-That is: as a stash of the same name with a single file of the same name; or, again, via a temporary file.
```

Comment:
Is there a reason you deleted this?

Reply:
Oh, that's me doing a poor job of splitting changes into two commits ;-)